l10n_vn_account_invoice_number/models/res_company.py

```diff
@@ -0,0 +1,13 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Trobz (http://trobz.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from odoo import api, fields, models, _
```

Comment:
Previous PR (https://github.com/fanha99/l10n-vietnam/pull/1) had some redundant imports: `api`, `_`

You can remove it in order to pass `travis-ci` checking :)